ocaml-lsp-server/src/document.ml

```diff
@@ -217,3 +217,14 @@ let get_impl_intf_counterparts uri =
     | to_switch_to -> to_switch_to
   in
   List.map ~f:Uri.of_path files_to_switch_to
+
+let edit doc range new_text =
```

Comment:
(nonessential)

Wdyt of a more explicit arg type 
```ocaml
new_text: [`Remove | `Replace_with of string] 

(* eg *) 
let _ = Document.edit doc range `Remove
```
?

Reply:
I probably wouldn't, since `Remove and `Replace_with do the same thing on empty strings anyway.